src/odemis/driver/rigol.py

```diff
@@ -441,7 +441,7 @@ def _decodeMessage(self, msg):
             except TypeError:   # could not unpack message
                 self._error_state = True
                 logging.exception('%s: Error Setting duty cycle %s', self.name,
-                                  cmd.decode('latin1', 'backslashreplace'))
+                                  msg.decode('latin1', 'backslashreplace'))
```

Comment:
I agree that it's more useful to use `msg` than `cmd` in case the command couldn't be parsed. That said, I don't think `msg.split()` should never fail due to TypeError as it's always the right type (bytes). I understand that during the migration you've hit such case, but now that it's fixed, there is no reason it'd happen again. This `except` was actually put there for the `float()`.

So the change looks good, but I'm not very convinced by the commit message. 
Could you apply the same change to the previous block (APPL:SQU) too?